src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter/InputWhitespaceAfterLambdaExpressions.java

```diff
@@ -11,19 +11,21 @@
 public class InputWhitespaceAfterLambdaExpressions {
 
     static Runnable r1 = () -> { // ok
-        String.valueOf("Hello world one!");
+        // String.valueOf("Hello world one!");
```

Comment:
Why is it commented out?

Reply:
done

Reply:
Not quite done, I assume that you have made changes on local, please push them to remote.

Reply:
Resolved